### DISCUSSION TITLE
How to fix "undisclosed-recipients@no.domain"?

### DISCUSSION THREAD

**Discussion by schleudertrauma** [jsuto/piler#196] [Q&A]
Hi there,

can anyone tell me how i could fix that mailpiler shows "undisclosed-recipients@no.domain" in the general search view in the 'To' bar?
The address showed in the 'From' section is correct.
I'm using mailcow and mails are sent via global transport map and sender/recipient-based bcc map to my mailpiler instance on 1.2.3.4:25
The 'from' and 'to' headers when i open the mail in piler are shown correctly. I don't really get it.

Piler:
1.4.6
Operating System
Linux piler 6.8.12-2-pve #1 SMP PREEMPT_DYNAMIC PMX 6.8.12-2 (2024-09-05T10:03Z) x86_64 x86_64 x86_64 GNU/Linux

Thanks in advance

**Comment by jsuto** [comment]
Probably you can't. The reason for this undisclosed recipients stuff is that both To: and Cc: headers fields are empty or missing from the email.
